Dockerfile.linux

```diff
@@ -29,4 +29,3 @@ WORKDIR /app/lisa
 
 RUN python3 -m pip install --no-cache-dir --upgrade pip && \
     python3 -m pip install --no-cache-dir --editable .[azure,libvirt,baremetal] --config-settings editable_mode=compat
```

Comment:
Add all tools like ado

Reply:
> Add ado

I don't understand this.

Reply:
Include the extra “ado” like azure, lobbies. All dev extra needs to be added.